diff -up gnutls-2.6.2/lib/gnutlsxx.cpp.nosrp gnutls-2.6.2/lib/gnutlsxx.cpp
--- gnutls-2.6.2/lib/gnutlsxx.cpp.nosrp	2008-11-12 10:56:28.000000000 +0100
+++ gnutls-2.6.2/lib/gnutlsxx.cpp	2008-12-04 14:34:10.000000000 +0100
@@ -396,12 +396,14 @@ void session::set_credentials( credentia
     RETWRAP(gnutls_credentials_set( s, cred.get_type(), cred.ptr()));
 }
 
-#ifdef ENABLE_SRP
 const char* server_session::get_srp_username() const
 {
+#ifdef ENABLE_SRP
     return gnutls_srp_server_get_username( s);
-}
+#else
+    return NULL;
 #endif
+}
 
 const char* server_session::get_psk_username() const
 {
